<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QueryViewSettingsWidget</class>
 <widget class="QWidget" name="QueryViewSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Query</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>6</number>
          </property>
          <property name="rightMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>6</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Maximum Rows</string>
              </property>
              <property name="buddy">
               <cstring>maxRowsSpinBox</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="maxRowsSpinBox">
              <property name="minimumSize">
               <size>
                <width>65</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>65</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="minimum">
               <number>10</number>
              </property>
              <property name="maximum">
               <number>30000</number>
              </property>
              <property name="singleStep">
               <number>10</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="showNullcheckBox">
              <property name="text">
               <string>Show '{null}' For NULL-Fields</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QCheckBox" name="sqlHistoryCheckBox">
              <property name="text">
               <string>Enable SQL History</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>View</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QCheckBox" name="showVerticalHeaderCheckBox">
            <property name="text">
             <string>Show Vertical Header</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Initial Row Height</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>defaultRowHeightSpinBox</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QSpinBox" name="defaultRowHeightSpinBox">
            <property name="minimumSize">
             <size>
              <width>65</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>65</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="minimum">
             <number>8</number>
            </property>
            <property name="maximum">
             <number>300</number>
            </property>
            <property name="singleStep">
             <number>1</number>
            </property>
            <property name="value">
             <number>14</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="resizeColumnsToContentsCheckBox">
            <property name="text">
             <string>Resize Columns To Contents</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Maximum Column Width</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>maxColumnWidthSpinBox</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QSpinBox" name="maxColumnWidthSpinBox">
            <property name="minimumSize">
             <size>
              <width>65</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>65</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="minimum">
             <number>120</number>
            </property>
            <property name="maximum">
             <number>600</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Clipboard</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Splitter Symbol</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="splitterLineEdit"/>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>130</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>16</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Font &amp;&amp; Colors</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QPushButton" name="fontPushButton">
         <property name="text">
          <string>Font</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QCheckBox" name="alternatingRowsCheckBox">
         <property name="text">
          <string>Alternating Rows</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Foreground Color</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QToolButton" name="color0ToolButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>91</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Background Colors (all/even rows)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QToolButton" name="color1ToolButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Background Colors (odd rows)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QToolButton" name="color2ToolButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>118</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
